pkg/k8s/apis/cilium.io/v2alpha1/bgpp_types.go

```diff
@@ -101,6 +105,46 @@ type CiliumBGPNeighborGracefulRestart struct {
 	RestartTimeSeconds *int32 `json:"restartTimeSeconds,omitempty"`
 }
 
+// BGPCommunityString holds a string containing a BGP Community Attribute value (RFC 1997)
+// as a 4-byte decimal number or two 2-byte decimal numbers separated by a colon,
+// or a BGP Large Community Attribute value (RFC 8092) as three 4-byte decimal numbers separated by a colon.
+// +kubebuilder:validation:Pattern=`^\d+(:\d+)?(:\d+)?$`
+type BGPCommunityString string
+
+// CiliumBGPPathAttributes can be used to apply additional path attributes
+// to matched routes when advertising them to a BGP peer.
+type CiliumBGPPathAttributes struct {
+	// SelectorType defines type of the object on which the Selector applies.
+	// - For "PodCIDR" the Selector matches k8s CiliumNode resources (resulting into PodCIDR routes).
+	// - For "CiliumLoadBalancerIPPool" the Selector matches CiliumLoadBalancerIPPool custom resources
+	//   (resulting into IP pool's CIDRs routes).
+	//
+	// +kubebuilder:validation:Enum=PodCIDR;CiliumLoadBalancerIPPool
+	// +kubebuilder:validation:Required
+	SelectorType string `json:"selectorType"`
+
+	// Selector selects a group of objects of the MatchType
```

Comment:
Should this be SelectorType instead of MatchType? Selector defines a label selector for matching objects of SelectorType type, correct?